src/core/layertree/qgslayertreemodel.h

```diff
@@ -382,6 +382,12 @@ class CORE_EXPORT QgsLayerTreeModel : public QAbstractItemModel
     void nodeLayerWillBeUnloaded();
     void layerLegendChanged();
 
+    /**
+     * Emitted when layer elevation properties have changed.
+     * \since QGIS 3.42
+     */
```

Comment:
This isn't a signal, so:

```suggestion
    /**
     * Triggered when layer elevation properties have changed.
     * \since QGIS 3.42
     */
```

Let's also move this to a private slot, it doesn't need to be exposed to Python.

Unrelated note -- the doc for `layerFlagsChanged` below is wrong, that also incorrectly states "Emitted" instead of "Triggered"

Reply:
Indeed, I did a copy-paste. My bad. I have added a commit to fix the `layerFlagsChanged` docstring. Fixed.